packages/cli-platform-android/src/commands/runAndroid/listAndroidTasks.ts

```diff
@@ -31,11 +31,14 @@ export const getGradleTasks = (
   taskType: 'install' | 'build',
   sourceDir: string,
 ) => {
+  const loader = getLoader();
+  loader.start('Searching for available Gradle tasks...');
   const cmd = process.platform.startsWith('win') ? 'gradlew.bat' : './gradlew';
 
   const out = execa.sync(cmd, ['tasks', '--group', taskType], {
     cwd: sourceDir,
   }).stdout;
+  loader.succeed();
```

Comment:
`execa.sync` can throw, can we catch the error, logger.fail and re-throw or handle error gracefully?